doc/TransmissionProtocol/TransmissionProtocol.tex

```diff
@@ -0,0 +1,354 @@
+also determine the direction, relative to the system, of that transmitter.
+The UTP broadcasts should be frequent enough for location updates to be
+practical to the user, while also being
+infrequent enough that the expected number of collisions is very low.
+For more information on collision statistics, see the document titled
+{\em RFC1\_stats.pdf}.
+
+Observe, however, that if every transmitter transmitted on a fixed interval,
+then we could theoretically encounter a situation in which two transmitters
+transmit at almost exactly the same time for the entire duration of their
+deployment.
+In other words, every single one of their transmissions would collide
+until one of the transmitters depleted its battery charge, and the receiver
+would never be able to reliably determine the location of either one.
+To remedy this, we will instead randomly vary the interval between broadcasts.
+That way, if two transmitters happen to transmit at the same time in one
+instance, they won't necessarily collide the second time around.
+The random adjustment will need to be recalculated for each interval in order
+to reduce our chances of collisions.
+
+The iCRAB protocol is expected to support only a finite number of transmitter
+UIDs.
+The upper bound selected is fairly arbitrary but based on the client's expected
+needs for the project's research.
+
+Finally, this protocol is also expected to be able to encode (along with the
+ID of a given transmitter) a boolean value.
+For the purposes of the project, this value will be \texttt{true}
+if and only if a given  transmitter is inert.
+This generally indicates that the crab died or molted its shell.
+We will encode this boolean value by effectively doubling the number of IDs
+that can be encoded.
+Half of the IDs are to be used when the boolean value is \texttt{false},
+and the other half are used when it is \texttt{true}.
+Specifically, values in the range $[0, MAX\_UID)$ are to be used when
+\texttt{inert=false}.
+There is no additional mapping when IDs are in this range.
+For all values in the range $[MAX\_UID, 2*MAX\_UID)$, the value encoded is
+actually $MAX\_UID$ bigger than the ID that is encoded, and
+\texttt{inert=true}.
+
+For example, if a signal encodes the value 42, the receiver should report
+this as transmitter 42 with \texttt{inert=false}.
+However, if the value 542 is received, assuming that $MAX\_UID=500$,
+the receiver should interpret this as transmitter 42 but with
+\texttt{inert=true}.
+
+For more background on the topic of boolean value encoding, see RFC 2.
+
+\subsection{Preamble to Definitions and Specifications} \label{preamble}
+
+Each Unique Transmission Pattern (UTP) will be formed by a ping, a delay,
+and another ping, in that order.
+Each transmitter will broadcast a UTP and will then wait for some interval
+before transmitting a UTP again.
+This process loops indefinitely throughout the transmitter's lifetime.
```

Comment:
Say more about "some interval". If the details are yet to be determined, say that.